packages/server/graphql/public/mutations/shareTopic.ts

```diff
@@ -0,0 +1,50 @@
+import {getUserId, isTeamMember} from '../../../utils/authorization'
+import standardError from '../../../utils/standardError'
+import {SlackNotifier} from '../../mutations/helpers/notifications/SlackNotifier'
+import {MutationResolvers} from '../resolverTypes'
+import DiscussStage from '../../../database/types/DiscussStage'
+
+const shareTopic: MutationResolvers['shareTopic'] = async (
+  _source,
+  {stageId, meetingId, channelId},
+  {authToken, dataLoader}
+) => {
+  const viewerId = getUserId(authToken)
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  const {teamId} = meeting
+
+  if (!isTeamMember(authToken, teamId)) {
+    return standardError(new Error('Team not found'), {userId: viewerId})
+  }
+
+  let stage = null
+  let stageIndex = -1
+
+  const discussPhase = meeting.phases.find((phase) => phase.phaseType === 'discuss')
```

Comment:
+1 you can use the getPhase helper here: https://github.com/ParabolInc/parabol/blob/ee91d7feb48aaab18e4a712af135db97700c60aa/packages/server/utils/getPhase.ts